app/code/core/Mage/Adminhtml/Model/Customer/Observer.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * OpenMage
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magento.com so we can send you a copy immediately.
+ *
+ * @category    Mage
+ * @package     Mage_Adminhtml
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (http://www.magento.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+
+/**
+ * Customer EAV Observer
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ * @author      Magento Core Team <core@magentocommerce.com>
+ */
+class Mage_Adminhtml_Model_Customer_Observer
+{
+
+    /**
+     * Add frontend properties to customer attribute edit form
+     *
+     * @param Varien_Event_Observer $observer
+     * @return $this
+     */
+    public function customerAttributeEditPrepareForm($observer)
+    {
+        /** @var Mage_Customer_Model_Attribute $attribute */
+        $attribute = $observer->getAttribute();
+
+        /** @var Varien_Data_Form $form */
+        $form = $observer->getForm();
+
+        /** @var Varien_Data_Form_Element_Fieldset $fieldset */
+        $fieldset = $form->getElement('base_fieldset');
+
+        // frontend properties fieldset
+        $fieldset = $form->addFieldset('front_fieldset', array('legend'=>Mage::helper('adminhtml')->__('Extra Properties')));
+
+        $fieldset->addField('use_in_forms', 'multiselect', array(
+            'name'   => 'use_in_forms',
+            'label'  => Mage::helper('adminhtml')->__('Use in Forms'),
+            'title'  => Mage::helper('adminhtml')->__('Use in Forms'),
+            'values' => Mage::getModel('customer/config_forms')->toOptionArray(),
+            'value'  => Mage::getResourceModel('customer/form_attribute')->getFormTypesByAttribute($attribute)
```

Comment:
Just looking at #2260 again, not sure why I implemented a new method instead of using `$attribute->getUsedInForms()`